Strangely the failing test is runingn out of disk space
`INFO     repo2docker:app.py:824 Error: committing container for step {Env:[NB_USER=runner NB_UID=1001 REPO_DIR=/srv/repo PATH=/home/runner/.local/bin:/srv/repo/.local/bin:/srv/conda/envs/notebook/bin:/srv/conda/bin:/srv/npm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin DEBIAN_FRONTEND=noninteractive LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 SHELL=/bin/bash USER=runner HOME=/home/runner APP_BASE=/srv CONDA_DIR=/srv/conda NB_PYTHON_PREFIX=/srv/conda/envs/notebook NPM_DIR=/srv/npm NPM_CONFIG_GLOBALCONFIG=/srv/npm/npmrc NB_ENVIRONMENT_FILE=/tmp/env/environment.lock MAMBA_ROOT_PREFIX=/srv/conda MAMBA_EXE=/srv/conda/bin/mamba CONDA_PLATFORM=linux-64 KERNEL_PYTHON_PREFIX=/srv/conda/envs/notebook CONDA_DEFAULT_ENV=/srv/conda/envs/notebook] Command:run Args:[TIMEFORMAT='time: %3R' bash -c 'time ${MAMBA_EXE} env update -p ${NB_PYTHON_PREFIX} --file "environment.yml" && time ${MAMBA_EXE} clean --all -f -y && ${MAMBA_EXE} list -p ${NB_PYTHON_PREFIX} '] Flags:[] Attrs:map[] Message:RUN TIMEFORMAT='time: %3R' bash -c 'time ${MAMBA_EXE} env update -p ${NB_PYTHON_PREFIX} --file "environment.yml" && time ${MAMBA_EXE} clean --all -f -y && ${MAMBA_EXE} list -p ${NB_PYTHON_PREFIX} ' Original:RUN TIMEFORMAT='time: %3R' bash -c 'time ${MAMBA_EXE} env update -p ${NB_PYTHON_PREFIX} --file "environment.yml" && time ${MAMBA_EXE} clean --all -f -y && ${MAMBA_EXE} list -p ${NB_PYTHON_PREFIX} '}: copying layers and metadata for container "c60d8adc11015838b3f6e819afdad9f6054ce954adf3118bb76ca8441f83c073": writing blob: adding layer with blob "sha256:c4b8ce47904a0e175bde526208956231b1dfb32611a0a98552d58c3b96036732": processing tar file(write /srv/conda/envs/notebook/libexec/gcc/x86_64-conda-linux-gnu/9.5.0/cc1obj: no space left on device): exit status 1`

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -84,10 +84,12 @@ jobs:
             python_version: "3.13"
             container_cli: podman
             repo_type: base
-          - ubuntu_version: "24.04"
-            python_version: "3.13"
-            container_cli: podman
-            repo_type: conda
+          # We are observing "no space left on device" on GitHub Actions,
+          # https://github.com/jupyterhub/repo2docker/pull/1471#issuecomment-3744990787
+          # - ubuntu_version: "24.04"
+          #   python_version: "3.13"
+          #   container_cli: podman
+          #   repo_type: conda
           - ubuntu_version: "24.04"
             python_version: "3.13"
             container_cli: podman

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -27,10 +27,13 @@ import pytest
 import requests
 import yaml
 
+import docker
+
 from repo2docker.__main__ import make_r2d
 
 TESTS_DIR = os.path.abspath(os.path.dirname(__file__))
 
+DOCKER_CLIENT = client = docker.from_env()
 
 def pytest_collect_file(parent, file_path):
     if file_path.name == "verify":
@@ -90,6 +93,12 @@ def make_test_func(args, skip_build=False, extra_run_kwargs=None, external_scrip
             # stop the container
             container.stop()
             app.wait_for_container(container)
+            try:
+                container.remove()
+            except:
+                pass
+            DOCKER_CLIENT.images.remove(image=app.output_image_spec)
+
 
     return test
 

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -28,12 +28,12 @@ import requests
 import yaml
 
 import docker
-
 from repo2docker.__main__ import make_r2d
 
 TESTS_DIR = os.path.abspath(os.path.dirname(__file__))
 
 DOCKER_CLIENT = client = docker.from_env()
+
 
 def pytest_collect_file(parent, file_path):
     if file_path.name == "verify":
@@ -98,7 +98,6 @@ def make_test_func(args, skip_build=False, extra_run_kwargs=None, external_scrip
             except:
                 pass
             DOCKER_CLIENT.images.remove(image=app.output_image_spec)
-
 
     return test
 

